test_package_validation.py

```
@@ -0,0 +1,254 @@
+#!/usr/bin/env python3
+"""
+Test script to validate package installation and functionality
+"""
+
+import subprocess
+import sys
+import os
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)